include/SDL_opengl_glext.h

```diff
@@ -1,5 +1,5 @@
-#ifndef __gl_glext_h_
-#define __gl_glext_h_ 1
+#ifndef __glext_h_
```

Comment:
Should SDL do the following instead? Or is OP doing something unsupported?
```c
#if !defined(__glext_h_) && !defined(__gl_glext_h_)
#define __glext_h_ 1
#define __gl_glext_h_ 1
```

Reply:
That might work. Tested?

Reply:
Just tested with:
```c
#include <SDL3/SDL.h>
#include <SDL3/SDL_opengl.h>
#include <SDL3/SDL_opengl_glext.h>

#include <GL/gl.h>
#include <GL/glext.h>

int main() {
 return 0;
}
```
and
```
#include <GL/gl.h>
#include <GL/glext.h>

#include <SDL3/SDL.h>
#include <SDL3/SDL_opengl.h>
#include <SDL3/SDL_opengl_glext.h>

int main() {
 return 0;
}
```
on my Linux system.

Reply:
Should be OK, then.